ncm-ceph/src/main/perl/Ceph/daemon.pm

```diff
@@ -208,6 +208,11 @@ sub check_empty {
             return 0;
         }
     } else {
+        my $checkmntcmd = ['/usr/bin/findmnt', $loc ];
```

Comment:
afaik, it is a standard package. But I've just seen that on el6 systems it is on a different path, /bin/findmnt..